MM-12290 inline image collapse (#4495)
mattermost/docs: source/cloud/cloud-administration/site-configuration.rst

File Sharing and Downloads
--------------------------

Control file sharing and downloads in your workspace and on mobile devices using the following three settings under **System Console > Site Configuration > File Sharing and Downloads**:

- **Allow File Sharing:** When configuration is set to **false**, file sharing is disabled. All file and image uploads on messages are forbidden across clients and devices, including mobile.
- **Allow File Uploads on Mobile:** When configuration is set to **false**, disables file uploads on mobile apps. All file and image uploads on messages are forbidden across clients and devices, including mobile.
- **Allow File Downloads on Mobile:** When configuration is set to **false**, disables file downloads on mobile apps. Users can still download files from a mobile web browser.

Select **Save** to apply your changes.

Localization
------------

Mattermost is translated into 18 languages. Users can set their default language by going to **Account Settings > Display > Language**, then selecting **Save**.

> New languages are added automatically by default. Users can add new languages using the dropdown menu manually as they become available. When manually adding new languages, the Default Client Language must be added before saving the setting.

Author: @esethna - Are 3 statements still true?

Reviewer: Yes afaik

Select **Save** to apply your changes.

Posts
-----

Control content in posts using the following four settings under **System Console > Site Configuration > Posts**:

- **Enable Link Previews:** Link previews are previews of linked website content, image links, and YouTube videos that are displayed below posts when available.

  - System Admins can disable all website link previews, image link previews, and YouTube previews by changing this setting to **false**. System Admins can also disable link previews from specific domains.
  - When link previews are enabled, users can choose to show or hide website link previews from **Account Settings > Display > Website Link Previews**. 
- **Enable SVGs:** Controls whether users have the ability to see previews of SVG file attachments and SVG image links.
- **Enable LaTeX Rendering:** Controls users' ability to render LaTeX code.
- **Custom URL Schemes:** A list of URL schemes that are used for autolinking in message text. HTTP, HTTPS, FTP, tel, and mailto always create links.

> In Mattermost Cloud, you can also control whether all `in-line images <https://docs.mattermost.com/help/messaging/formatting-text.html#in-line-images>`__ over 100px in height are automatically collapsed or expanded in messages by setting a `user preference <https://docs.mattermost.com/help/settings/account-settings.html#link-previews>`__, or by using the ``/collapse`` and ``/expand`` `slash commands <https://docs.mattermost.com/cloud/cloud-integrations/cloud-slash-commands.html>`__. 

Author: ```suggestion
You can also control whether all `in-line images <https://docs.mattermost.com/help/messaging/formatting-text.html#in-line-images>`__ over 100px in height are automatically collapsed or expanded in messages by setting a `user preference <https://docs.mattermost.com/help/settings/account-settings.html#link-previews>`__, or by using the ``/collapse`` and ``/expand`` `slash commands <https://docs.mattermost.com/cloud/cloud-integrations/cloud-slash-commands.html>`__. 
```

> From Mattermost Server v5.35, you can also control whether all `in-line images <https://docs.mattermost.com/help/messaging/formatting-text.html#in-line-images>`__ over 100px in height are automatically collapsed or expanded in messages by setting a `user preference <https://docs.mattermost.com/help/settings/account-settings.html#link-previews>`__, or by using the ``/collapse`` and ``/expand`` `slash commands <https://docs.mattermost.com/cloud/cloud-integrations/cloud-slash-commands.html>`__. 

Author: ```suggestion
In Mattermost Cloud, you can also control whether all `in-line images <https://docs.mattermost.com/help/messaging/formatting-text.html#in-line-images>`__ over 100px in height are automatically collapsed or expanded in messages by setting a `user preference <https://docs.mattermost.com/help/settings/account-settings.html#link-previews>`__, or by using the ``/collapse`` and ``/expand`` `slash commands <https://docs.mattermost.com/cloud/cloud-integrations/cloud-slash-commands.html>`__. 
```

> From Mattermost Server v5.35, you can also control whether all `in-line images <https://docs.mattermost.com/help/messaging/formatting-text.html#in-line-images>`__ over 100px in height are automatically collapsed or expanded in messages by setting a `user preference <https://docs.mattermost.com/help/settings/account-settings.html#link-previews>`__, or by using the ``/collapse`` and ``/expand`` `slash commands <https://docs.mattermost.com/cloud/cloud-integrations/cloud-slash-commands.html>`__. 

Author: ```suggestion
You can also control whether all `in-line images <https://docs.mattermost.com/help/messaging/formatting-text.html#in-line-images>`__ over 100px in height are automatically collapsed or expanded in messages by setting a `user preference <https://docs.mattermost.com/help/settings/account-settings.html#link-previews>`__, or by using the ``/collapse`` and ``/expand`` `slash commands <https://docs.mattermost.com/cloud/cloud-integrations/cloud-slash-commands.html>`__. 
```

Select **Save** to apply your changes.

Users and Teams
---------------

> Control the following users and teams ettings under **System Console > Site Configuration > Users and Teams**:

Author: ```suggestion
Control the following users and teams settings under **System Console > Site Configuration > Users and Teams**:
```

  - A team of people is considered a small organization where people work closely together towards a specific shared goal and share the same etiquette. 
  - In the physical world, a team of people could typically be seated around a single table to have a meal and discuss their project. 
  - The default maximum of 1000 people is at the extreme high end of a single team of people where organizations are more often 'multiple teams of people'. Investments in explicitly defining etiquette, such as channel organization or turning on policy features in Enterprise Edition, are often used to scale the high levels of productivity found in a team of people using Mattermost to multiple teams of people.
- **Max Channels Per Team:** Set the maximum number of channels per team, including both active and deleted channels.
- **Enable users to open Direct Message channels with:** Configure whether users can message any user on the Mattermost workspace or only users in the same team as them.

  - This setting adjusts the users returned in the Direct Messages **More** menu. The CTRL/CMD+K channel switcher only lists users on the current team. 

>   - This setting only affects the UI, not permissions on the server. For instance, a Direct Message channel can be created with anyone on the server regardless of this setting.

Reviewer 2: ```suggestion
  - This setting only affects the user interface, not permissions on the server. For instance, a Direct Message channel can be created with anyone on the server regardless of this setting.
```

- **Teammate Name Display:** Specify how names are displayed in the user interface by default. Please note that users can override this setting in **Account Settings > Display > Teammate Name Display**.

  - **Show username** displays the user's username. 
  - **Show nickname** if one exists displays the user's nickname. If the user does not have a nickname, their full name is displayed. If the user does not have a full name, their username is displayed. 

>   - **Show first and last name** displays the user's full name. If the user does not have a full name, their username is displayed. Recommended when using SAML or LDAP if first name and last name attributes are configured.  

Reviewer 2: ```suggestion
  - **Show first and last name:** displays the user's full name. If the user doesn't have a full name, their username is displayed. Recommended when using SAML or LDAP if first name and last name attributes are configured.
```

- **Lock Teammate Name Display for all users:** Specifies whether users can change settings under **Main Menu > Account Settings > Display > Teammate Name Display**.

> - **Show Email Address:** When set to **false**, this setting hides email address of users from other users in the user interface, including Team Admins. This is designed for managing teams where users choose to keep their contact information private. System Admins will still be able to see email addresses in the UI.

Reviewer 2: ```suggestion
- **Show Email Address:** When set to **false**, this setting hides a user's email address from other users, including Team Admins. This is designed for managing teams where users choose to keep their contact information private. System Admins will still be able to see email addresses in the user interface.
```

> - **Show Full Name:** When set to false, this setting hides full name of users from other users including Team Admins. This is designed for managing teams where users choose to keep their contact information private. System Admins will still be able to see full names in the UI.

Reviewer 2: ```suggestion
- **Show Full Name:** When set to **false**, this setting hides full name of users from other users including Team Admins. This is designed for managing teams where users choose to keep their contact information private. System Admins will still be able to see full names in the user interface.
```
